src/main/java/uk/gov/companieshouse/service/dissolution/chips/ChipsResponseService.java

```diff
@@ -48,13 +49,19 @@ public void saveAndNotifyDissolutionApplicationOutcome(ChipsResponseCreateReques
         dissolution.setActive(false);
 
         if (featureToggleConfig.isRefundsEnabled() && dissolutionVerdict.getResult() == VerdictResult.REJECTED) {
-            dissolutionRefundService.handleRefund(dissolution);
+            try {
+                dissolutionRefundService.handleRefund(dissolution);
+            } catch (WebClientException e) {
+                logger.info(String.format("Automatic refund failed, sending rejection email to finance. Exception: %s", e.getMessage()));
+
+                dissolutionEmailService.sendApplicationOutcomeEmail(dissolution, dissolutionVerdict, true);
+            }
```

Comment:
i believe the dissolution refund service should handle this. It is specific to dissolution refunds rather than the wider chips aspect.

Also I suggest you make the log a warning so we can easily identify failed refunds.